build.gradle

```diff
@@ -103,14 +103,15 @@ ext {
             rekognition: "com.amazonaws:aws-android-sdk-rekognition:$awsSdkVersion",
             textract: "com.amazonaws:aws-android-sdk-textract:$awsSdkVersion",
             translate: "com.amazonaws:aws-android-sdk-translate:$awsSdkVersion",
-            
+
             cognitoidentity: "aws.sdk.kotlin:cognitoidentity:$awsKoltinSdkVersion",
             cognitoidentityprovider: "aws.sdk.kotlin:cognitoidentityprovider:$awsKoltinSdkVersion",
             credentials: "aws.smithy.kotlin:aws-credentials:0.10.2",
             ktor: "aws.smithy.kotlin:http-client-engine-ktor:0.7.7",
             location: "aws.sdk.kotlin:location:$awsKoltinSdkVersion",
             s3: "aws.sdk.kotlin:s3:$awsKoltinSdkVersion",
             signing: "aws.smithy.kotlin:aws-signing-default:0.10.2",
+            pinpointKotlin: "aws.sdk.kotlin:pinpoint:$awsKoltinSdkVersion"
```

Comment:
I know this isn't related to this PR, but can we just fix `Koltin` -> `Kotlin` one of these days?  Preferably before GA 😄